README.md

```diff
@@ -1 +1,12 @@
-# pulumi-grid
\ No newline at end of file
+# pulumi-grid
```

Comment:
let's make a structure for examples:
examples dir -> network_example dir -> Pulumi.yaml